Interface/AddOns/nMainbar/config.lua

```diff
@@ -45,6 +45,12 @@ nMainbar.Config = {
         font = 'Fonts\\ARIALN.ttf',
     },
 
+    artifactBar = {
+        mouseover = true,
```

Comment:
Is this a good default? I'm thinking if it's something that you use frequently, it might be better to default to `false`? It also seems like people have a hard time finding stuff hidden by default, especially when they kind of have a weird default position (like in a corner or something).